### PR TITLE
Desktop view

### PR DESCRIPTION
**Pull request by EbukaNweje**
### Description
I use media queries to make sure that my website displays correctly on desktop screens.

- I make sure that my desktop version looks like the one in the Figma design template that I chose. I include a screenshot of Template 1 below for reference, but you should use the template that you chose.

### Project requirements
- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).

- In order to lay out the elements on the page you should use Grid and Flexbox.

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @EbukaNweje,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request**

You have submitted a project, that _**Your PR doesn't have descriptive details about the project, please add details and re-submit**_

Capture web_2-6-2022_10527_github com
Capture web_2-6-2022_105310_gitlab com


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -138,8 +158,13 @@ <h4>Multi-Post
                     A daily selection of privately personalized reads; 
                     no accounts or sign-ups required.
                 </p>
+                <p class="Info2">
+                    A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
```
Comment (tufoinnkuo10) [inline index.html:164]: - [ ] please make the text section of your cards responsive when switching from the mobile version to the desktop version or vice versa. 
your version.
Screenshot (101)_LI

File: index.html [hunk 2]
```diff
@@ -13,12 +13,17 @@
    <nav>
      <a href=""><h4>My Logo</h4></a>
      <img src="./img/Menu.png" alt="">
+     <ul>
+        <li><a href="#">Portoflio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+     </ul>
```
Comment (tufoinnkuo10) [inline index.html:20]: - [ ] kindly add a little left and right margin to your navigation bar..

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @EbukaNweje 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ app is responsive

Required Changes ♻️
Check the comments under the review

tufoin
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ishimwezachee — APPROVED** [review]
Hi @EbukaNweje , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**